javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,110 @@
+let userName = ""; // Rodrigo;
+const todoList = [];
+let lettersToCount = [];
+
+function getReply(command) {
```

Comment:
A general comment to this function is that it is way to big and handles to much logic. The reason this is not good is for code maintainability and readability. Therefore what i would advise you to do is to split this big function into smaller ones.